packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx

```diff
@@ -156,14 +156,18 @@ const TeamTasksHeader = (props: Props) => {
       </TeamHeaderAndAvatars>
       <DashSectionControls>
         {/* Filter by Owner */}
-        <DashFilterToggle
-          label='Team Member'
-          onClick={togglePortal}
-          onMouseEnter={TeamDashTeamMemberMenu.preload}
-          ref={originRef}
-          value={teamMemberFilterName}
-        />
-        {menuPortal(<TeamDashTeamMemberMenu menuProps={menuProps} team={team} />)}
+        <TeamMeta>
+          {teamMembers.length && (
```

Comment:
The condition should probably be `teamMembers.length` > 1 as I still able to see the filter with one member in the team

<img width="1524" alt="image" src="https://user-images.githubusercontent.com/466991/196958914-0f99150e-b3b6-4be2-affd-949c787f6167.png">